src/flureeQuery.js

```diff
@@ -47,9 +47,11 @@ function flureeQuery(query, opts) {
         get: getResult()
     });
 
-     // imperfect but fast comparison values for useEffect() to determine if we need to re-run query - false positive is OK
+    // imperfect but fast comparison values for useEffect() to determine if we need to re-run query - false positive is OK
     const useEffectCmp = [JSON.stringify(query)]
 
+    console.log("useEffectCmp", useEffectCmp);
```

Comment:
A leftover debug log?

Reply:
Yes. Trying to figure out why the multi query didn't work. I'll take that out before merging